### PR TITLE
Rename QoS*Policy enum's to *Policy

### PR DESCRIPTION
**Pull request by suab321321**
…ters

Signed-off-by: Abhinav <singhabhinav9051571833@gmail.com>

----
Edit(@ivanpauno):

Fixes https://github.com/ros2/rclpy/issues/371

### CODE REVIEW

File: rclpy/rclpy/qos.py [hunk 1]
```diff
@@ -270,6 +270,9 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#Alias with the old name, for retrocompatibility
+HistoryPolicy = QoSHistoryPolicy
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]:
```python3
QoSHistoryPolicy = HistoryPolicy
```
And change the class name from `QosHistoryPolicy` to `HistoryPolicy`.

I know that it looks like the same, but in this way the old name can be easily deprecated/deleted later.

File: rclpy/rclpy/qos.py [hunk 2]
```diff
@@ -285,6 +288,9 @@ class QoSReliabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT = 2
     BEST_EFFORT = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT
 
+#Alias with the old name, for retrocompatibility
+ReliabilityPolicy = QoSReliabilityPolicy
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: same

File: rclpy/rclpy/qos.py [hunk 3]
```diff
@@ -300,6 +306,9 @@ class QoSDurabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_DURABILITY_VOLATILE = 2
     VOLATILE = RMW_QOS_POLICY_DURABILITY_VOLATILE
 
+#Alias with the old name, for retrocompatibility
+DurabilityPolicy = QoSDurabilityPolicy
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: same

File: rclpy/rclpy/qos.py [hunk 4]
```diff
@@ -317,6 +326,9 @@ class QoSLivelinessPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC = 3
     MANUAL_BY_TOPIC = RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC
 
+#Alias with the old name, for retrocompatibility
+LivelinessPolicy = QoSLivelinessPolicy
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: same

**Comment by ivanpauno** [comment]
For the record, this is a follow-up of https://github.com/ros2/rclpy/pull/376.

**Comment by ivanpauno** [comment]
Fixes https://github.com/ros2/rclpy/issues/371.

File: rclpy/rclpy/qos.py (hunk 1 above)
Comment (suab321321) [reply rclpy/rclpy/qos.py]: extremly sorry for this silly mistake..

**Comment by suab321321** [comment]
I will update the changes

File: rclpy/rclpy/qos.py (hunk 1 above)
Comment (suab321321) [reply rclpy/rclpy/qos.py]: I have made the changes required..please have a look to those

**Comment by suab321321** [comment]
I want to ask one more thing ...
Is it necessary to understand the whole codebase of a open source project to start contributing to some tough issues?
Thank you

File: rclpy/rclpy/qos.py [hunk 5]
```diff
@@ -270,6 +270,9 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#Alias with the old name, for retrocompatibility
+QoSHistoryPolicy = HistoryPolicy 
+
 
 class QoSReliabilityPolicy(QoSPolicyEnum):
```
Comment (ivanpauno) [inline rclpy/rclpy/qos.py]: You didn't renamed the classes, this should be `ReliabilityPolicy`.
Same for the other three cases (L259, L295, L313).

File: rclpy/rclpy/qos.py (hunk 1 above)
Comment (ivanpauno) [reply rclpy/rclpy/qos.py]: You don't have to sorry, it's ok to make mistakes.

**Comment by ivanpauno** [comment]
> Is it necessary to understand the whole codebase of a open source project to start contributing to some tough issues?

I really don't understand the whole codebase. But you have to thoroughly understand the issue.

File: rclpy/rclpy/qos.py (hunk 5 above)
Comment (suab321321) [reply rclpy/rclpy/qos.py]: Done..

**Comment by ivanpauno** [comment]
Sanity check:
- Linux: [Build Status](https://ci.ros2.org/job/ci_linux/7448/)

**Comment by sloretz** [comment]
Thanks for the PR @suab321321 !

**Pull request merged**